### PR TITLE
Feature/10 log events

### PR DESCRIPTION
**Pull request by jschirrmacher**
Events will be logged only if hosted somewhere different than GitHub pages on a server or docker container running Node.js.

The small Node.js based server only serves the plugin files, and injects different URLs to the `manifest.xml` and `message.js` files to enable logging to a loggia-Host.

All URLs are given as environment variables to the Node.js server:

    LOGGER_URL=https://internal.server.running.loggia
    INTERNAL_PLUGIN_URL=https://internal.server.running.index.js
    PUBLIC_PLUGIN_URL=https://assistify.github.io/outlook-plugin

Optionally, you can use `DEBUG=true` to log all request to the console

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const logger = console
+
+const PORT = process.env.PORT || 3000
+const app = express()
+
+const messagejs = fs.readFileSync(__dirname + '/src/helper/message.js').toString()
+  .replace(/^var usageLogger = null;/, `var usageLogger='${process.env.LOGGER_URL}';`)
+
+const manifestxml = fs.readFileSync(__dirname + '/manifest.xml').toString()
+  .replace(new RegExp(process.env.PUBLIC_PLUGIN_URL, 'g'), process.env.INTERNAL_PLUGIN_URL)
```
Comment (ThomasRoehl) [inline index.js:12]: This is pretty dirty but seems to work.

**Review by ThomasRoehl — CHANGES_REQUESTED** [review]
We need a section in the readme for that.
After that it looks ok to merge

**Comment by vickyokrm** [comment]
@jschirrmacher Do we need this PR?

**Comment by jschirrmacher** [comment]
Not anymore.

**Pull request closed without merging**